### PR TITLE
OCNE on Azure or vSphere

### PR DESCRIPTION
**Pull request by rhomast**

### CODE REVIEW

File: content/en/docs/advanced/ocne-vsphere.md [hunk 1]
```diff
@@ -0,0 +1,1792 @@
...
+After the cluster resources are created, you must perform some additional steps to finish the configuration of the cluster.
+
+1. If vSphere does not have a load-balancer, then you can deploy MetalLB.
+{{< clipboard >}}
+<div class="highlight">
+
+```
+$ export KUBECONFIG=kluster1
+ 
+ADDRESS_RANGE=${1:-"subnet-from-vSphere-network"};
```
Comment (rhomast) [inline content/en/docs/advanced/ocne-vsphere.md:1648]: Should the be an `export` in front of this?

**Review by rhomast — COMMENTED** [review]
Please look over the env variables and their values carefully. I tried to anonymize the various values that were clearly user/environment specific but I might have missed one or anonymized one that was hardcoded

File: content/en/docs/advanced/ocne-vsphere.md [hunk 2]
```diff
@@ -0,0 +1,1792 @@
...
+1. Ensure that the various components were created successfully. Follow the instructions at [Monitor the SDDC Creation Process](https://docs.oracle.com/en/solutions/deploy-vmware-sddc-oci/deploy-sddc-cloud1.html#GUID-E170AD8F-7E51-44C6-93D9-DC332F3D8025) in the Oracle Help Architecture Center.
+
+
+## Prepare the VM environment
+
+1. Download an Oracle Linux 8 ISO image from [Oracle Linux Installation Media](https://yum.oracle.com/oracle-linux-isos.html).
+
+1. Deploy a VM by following the instructions at [Create a Virtual Machine with the New Virtual Machine Wizard](https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.vm_admin.doc/GUID-AE8AFBF1-75D1-4172-988C-378C35C9FAF2.html?hWord=N4IghgNiBcIEZgM4FMAEA3AtiAvkA) in the vSphere documentation.
+1. Upload the Oracle Linux 8 ISO image to vSphere. Use the steps at [Upload ISO Image Installation Media for a Guest Operating System](https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.vm_admin.doc/GUID-492D6904-7471-4D66-9555-9466CCCA6931.html) in the vSphere documentation.
```
Comment (theabmitra) [inline content/en/docs/advanced/ocne-vsphere.md:51]: Upload ISO should come before Deploy a VM step

File: content/en/docs/advanced/ocne-vsphere.md [hunk 3]
```diff
@@ -0,0 +1,1792 @@
...
+## Set up the admin cluster
+
+The Cluster API requires an initial cluster as a starting point to deploy its resources.
+
+1. Install kind. Follow the instructions at [Installation](https://kind.sigs.k8s.io/docs/user/quick-start/#installation) in the kind documentation.
+
+1. Create a Kubernetes cluster using kind. This cluster must be accessible by the VMware SDDC. Follow the instructions at [Quick Start: Install and/or configure a Kubernetes cluster](https://cluster-api.sigs.k8s.io/user/quick-start#install-andor-configure-a-kubernetes-cluster) in The Cluster API Book.
+1. Install the clusterctl CLI tool. clusterctl manages the lifecycle operations of a cluster API admin cluster. Follow instructions at [Quick Start: Install clusterctl](https://cluster-api.sigs.k8s.io/user/quick-start.html#install-clusterctl) in the Cluster API Book.
+1. Install the Verrazzano CLI tool using the instructions at [CLI Setup]({{< relref "/docs/setup/install/prepare/cli-setup" >}}).
+1. Install Verrazzano on the cluster using either the `dev` or `prod` profile. Follow the instructions at [Install with CLI]({{< relref "/docs/setup/install/perform/cli-installation" >}}).
```
Comment (theabmitra) [inline content/en/docs/advanced/ocne-vsphere.md:91]: Specific step to install verrazzano would be helpful

File: content/en/docs/advanced/ocne-vsphere.md (hunk 1 above)
Comment (theabmitra) [reply content/en/docs/advanced/ocne-vsphere.md:1648]: no this is fine

File: content/en/docs/advanced/ocne-vsphere.md (hunk 3 above)
Comment (rhomast) [reply content/en/docs/advanced/ocne-vsphere.md:91]: There's a link to the Installation steps at the end of the step.

**Review by bryantidd — APPROVED** [review]

File: content/en/docs/advanced/ocne-vsphere.md [hunk 4]
```diff
@@ -0,0 +1,1792 @@
...
+{{< clipboard >}}
+<div class="highlight">
+
+```
+$ clusterctl init -n verrazzano-capi -i vsphere
+```
+</div>
+{{< /clipboard >}}
+
+clusterctl will report when the admin cluster was successfully initialized.
```
Comment (rosemarymarano) [inline content/en/docs/advanced/ocne-vsphere.md:124]: -> clusterctl will report when the admin cluster was successfully initialized. (Indent this sentence to be under the preceding step.)

File: content/en/docs/advanced/ocne-vsphere.md [hunk 5]
```diff
@@ -0,0 +1,1792 @@
...
+$ clusterctl get kubeconfig kluster1 -n kluster1 > kluster1
+```
+</div>
+{{< /clipboard >}}
+
+## Finish cluster configuration 
+
+After the cluster resources are created, you must perform some additional steps to finish the configuration of the cluster.
+
+1. If vSphere does not have a load-balancer, then you can deploy MetalLB.
```
Comment (rosemarymarano) [inline content/en/docs/advanced/ocne-vsphere.md:1641]: -> load balancer (globally)

**Review by rosemarymarano — COMMENTED** [review]
Minor text edits.

**Review by rosemarymarano — APPROVED** [review]

**Pull request merged**